packages/react-storage/src/components/StorageBrowser/context/elements/defaults.tsx

```diff
@@ -125,6 +126,14 @@ function Heading(props: HeadingElementProps): React.JSX.Element {
   );
 }
 
+function Span(props: SpanElementProps): React.JSX.Element {
+  const { variant } = props;
+  if (variant === 'navigate-current') {
+    return <_View {...props} as="span" padding="xs" color="font.secondary" />;
```

Comment:
The font size of the "current" item in the Amplify screenshot looks a bit larger than the button text, do we need to add anything here?

Reply:
Good catch, fixed that in latest.